src/VisualStudio/TestUtilities/OutOfProcess/SolutionExplorer_OutOfProc.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Roslyn.VisualStudio.Test.Utilities.InProcess;
+
+namespace Roslyn.VisualStudio.Test.Utilities.OutOfProcess
+{
+    public class SolutionExplorer_OutOfProc : OutOfProcComponent<SolutionExplorer_InProc>
+    {
+        public SolutionExplorer_OutOfProc(VisualStudioInstance visualStudioInstance)
+            : base(visualStudioInstance)
+        {
+        }
+
+        public void CloseSolution(bool saveFirst = false) => InProc.CloseSolution(saveFirst);
+
+        /// <summary>
+        /// Creates and loads a new solution in the host process, optionally saving the existing solution if one exists.
+        /// </summary>
+        public void CreateSolution(string solutionName, bool saveExistingSolutionIfExists = false)
```

Comment:
Sure. I think I only different because the parameter lists mostly got longer. I actually dislike expression bodies for void methods. I'll fix the one above.